drivers/hmis/app/graphql/schema.graphql

```diff
@@ -8,7 +8,7 @@ type Client {
   Date of birth as format yyyy-mm-dd
   """
   dob: ISO8601Date
-  enrollments: [Enrollment!]!
+  enrollments(limit: Int, offset: Int): EnrollmentsPaginated!
```

Comment:
We are going to need `sortOrder: MOST_RECENT` here that sorts by `EntryDate` (I think–– should confirm with @eanders because in the Warehouse we sort by `first_date_in_program` which is a derived field on ServiceHistoryEnrollment, and I'm not sure if it's different from EntryDate).

Up to you whether you want to do it on this branch or a follow-up, I that should be all we need to close out #182857429!

Reply:
I'll go ahead and do that here

Reply:
Done

Reply:
Testing this out, I'm getting this error with the following query:
```
- `sort_order: most_recent` was given by GraphQL but not defined in the Ruby method. Add `sort_order:` to the method parameters.
```

```
{
  client(id: "409") {
    id
    firstName
    enrollments(limit: 2, sortOrder: MOST_RECENT) {
      nodesCount
      nodes {
        id
        project {
          projectName
        }
        entryDate
        exitDate
      }
    }
  }
}
```

Reply:
Alright, this is fixed as discussed